jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -224,7 +224,7 @@ def update(
         ...,
         help="The id of the job script to update",
     ),
-    name: str = typer.Option(
+    name: Optional[str] = typer.Option(
```

Comment:
Use modern type hinting here:
```suggestion
    name: str | None = typer.Option(
```

Reply:
It is valid if we add `from __future__ import annotations`, but it still results in some internal errors in `typer`, so reverting the suggestion.